src/TaskListItem.cpp

```diff
@@ -135,7 +135,12 @@ TaskListItem::GetStringTime()
 	Hours = (m_SpentTime / 60 / 60) % 24;
 	Day = (m_SpentTime / 60 / 60 / 24);
 
-	sprintf(buffer, "%d days, %d:%d:%d", Day, Hours, Minutes, Seconds);
+	if (Day==0)
+		sprintf(buffer, "%d:%d:%d", Hours, Minutes, Seconds);
+	else if (Day==1)
```

Comment:
Such checks are not appropriate for any language other than english. For example, in Russian: https://github.com/HaikuArchives/TimeTracker/pull/17 there are different rules for 2 to 4, than for 5 and up.

I think the best thing to do here is to use a BDurationFormat, which will do the right thing for you. For an example use, you can check AboutSystem in Haiku, which uses this to show the system uptime.